### ISSUE TITLE
examples/tree-example.sh defunct

### ISSUE THREAD

**Issue by chiarcos** [acoli-repo/conll-rdf#34]
I found that tree-example.sh doesn't return any triples.

**Comment by glaserL** [comment]
@cfaeth I think I had this issue before I can handle it

**Comment by glaserL** [comment]
@chiarcos I would need more infos of configuration to replicate, tree-example.sh works fine on ubuntu 16.04 and macos 10.15. I remember when I had this issue the script was missing -custom parameter when calling CoNLLRDFUpdater, could that help?

best 
luis glaser

**Comment by chiarcos** [comment]
Indeed, here it is the CoNLLRDFUpdater eating up its input.  CoNLLBrackets2RDF seems ok. 
However, the call

$> $ROOT/run.sh CoNLLRDFUpdater -custom -updates $SPARQL/trees/tree2bracket.sparql

looks just fine.

I replaced  line 26 with 

$> $ROOT/run.sh CoNLLRDFUpdater

but even then, it only spelled out its prefix declarations (it should just return its input).

Could that have something to do with the revised behavior on @prefixes?

**Comment by chiarcos** [comment]
Sorry, I just realized that was due to a local modification.

I had the following changes in CoNLLBrackets2RDF, and somehow, these cause that error. However, as I don't see how that made CoNLLRDFUpdater (!) fail, so, there might still be an issue with the Updater here.

l.60
<                                               for(int i = 0; i<fields.length; i++)
>                                               for(int i = 0; i<fields.length && i<col2bracket.length; i++)
l.96
<                                       if(i<this.col2bracket.length && this.col2bracket[i]) 
>                                       if(i<this.col2bracket.length && this.col2bracket[i]!=null) { 

Both changes were necessary to address another issue with CoNLLBrackets2RDF, I create a separate issue for that.
